share/spice/tvmaze/nextepisode/tvmaze_nextepisode.js

```diff
@@ -0,0 +1,39 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_tvmaze_nextepisode = function(api_result){
+        if (!api_result || !api_result._embedded) {
+            return Spice.failed('tvmaze');
```

Comment:
these need to match the Perl package/callback name: `Spice.failed('tvmaze_nextepisode');`